src/geoserver-publish-imagemosaic/package.json

```diff
@@ -10,7 +10,9 @@
   },
   "dependencies": {
     "amqplib": "^0.8.0",
-    "geoserver-node-client": "1.1.0"
+    "fs.promises": "^0.1.2",
+    "geoserver-node-client": "1.1.0",
+    "path": "^0.12.7"
```

Comment:
just checked this package. It is 7 years old. see https://www.npmjs.com/package/path
I am pretty sure we can perform the same operation with build-in functionality from Node.js